playground/README.md

```diff
@@ -8,3 +8,26 @@
 This is a Custom Application used to help developing features in this repository.
 
 Please read the [documentation](https://docs.commercetools.com/custom-applications) about Custom Applications before getting started.
+
+## Using the Echo Server view
```

Comment:
Can we maybe split the changes into two PRs? One for the fix with the polyfill, the other with the improvements to the playground.

Reply:
Ok. I used different commits to show both actions, but I will create a new PR for the playground and remove those changes from this one.

Reply:
Remove the playground part from this PR in this commit: 8ade2b4e51adba4bf2a0d4f207c19d46b5e3effa